taskcluster/scriptworker-task-github.yml

```diff
@@ -33,5 +33,5 @@ build:
       - "linux-rpi3-cpu-opt"
       - "linux-arm64-cpu-opt"
   metadata:
-    name: "DeepSpeech Packages"
-    description: "Trigger Uploading of DeepSpeech Packages to the Internets"
+    name: "DeepSpeech GitHub Packages"
```

Comment:
This task is going to need a few changes on ScriptWorker side so that we can perform the upload only to GitHub